### PR TITLE
Ukrainian language support

### PR DESCRIPTION
**Pull request by spalax**
Hi i've added ukrainian language to your library, will be greate if you will check and merge my pull request.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+3.2.0 / 2015-07-15
```
Comment (EvanHahn) [inline HISTORY.md]: Could you change this header to "unreleased"? It may not be released today.

**Comment by EvanHahn** [comment]
This looks great!

A couple of things:

1. I _just_ made a change to the readme that makes a merge conflict—sorry about that. Could you merge with `master`?
1. `getUkrainianForm` seems the same as `getRussianForm`. Could you rename that function and just use one?
1. I left a small comment in `HISTORY.md`.

Other than that, this is perfect!

**Comment by spalax** [comment]
I saw that functions are the same, but in my head now do not born good naming which will unite  Ukrainian and Russian forms together. 
I found https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals and i guess to name function with link :
```
// Internal helper function for (Belarusian, Bosnian, Croatian, Serbian, Russian, Ukrainian) languages.
// plese check https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals
// to understand the what the Rule 7 means.
    function getSlavicPluralRule7Form(c) {
```

What do you think ?

**Comment by spalax** [comment]
Or just getSlavicForm , but Polish it is Slavic to.

**Comment by spalax** [comment]
I've merged, and fixed mistakes.

**Comment by EvanHahn** [comment]
_Дякую_! I'll push a new version soon.

**Comment by EvanHahn** [comment]
Pushed `humanize-duration@3.2.0`!

**Pull request merged**